.github/workflows/c-cpp.yml

```diff
@@ -1,19 +0,0 @@
-name: C/C++ CI
```

Comment:
I don't think this was supposed to happen, can you add this file back to the branch?